### PR TITLE
texi: un-comment indices.

### PR DESCRIPTION
**Pull request by HugoNikanor**
I see no reason for not rendering the index that already is there.

The concept index is currently also empty, but I enabled it to in case something gets added to it later.

### CODE REVIEW

File: fibers.texi [hunk 1]
```diff
@@ -1389,11 +1389,11 @@ Guile now has a solid concurrency story.  Use fibers, incorporate it
...
 up to you.  Happy hacking and godspeed!
 
-@c @node Concept Index
-@c @unnumbered Concept Index
-@c @printindex cp
-@c @node Function Index
-@c @unnumbered Function Index
-@c @printindex fn
+@node Concept Index
+@unnumbered Concept Index
```
Comment (emixa-d) [inline fibers.texi:1393]: I think including an index while it's still empty is a bit silly, but IMO go ahead with the rest!
I would like some confirmation from another reviewer on the location though.

**Review by emixa-d — CHANGES_REQUESTED** [review]

File: fibers.texi (hunk 1 above)
Comment (emixa-d) [reply fibers.texi:1393]: (it can easily be re-added once there's actually some things inside)

**Comment by HugoNikanor** [comment]
I re-commented the unused concept index.

-- 
Hugo Hörnquist

**Comment by emixa-d** [comment]
Looks like there are some errors, according to the CI, could you fix them?

**Comment by HugoNikanor** [comment]
I couldn't recreate the warning locally, but hopefully this fixes it.

**Comment by emixa-d** [comment]
I've rebased it in <https://github.com/wingo/fibers/pull/95>. Assuming no negative review (and assuming I don't forget), I intend to merge it after 5 days.

**Pull request merged**